src/core/best-practices.md

```diff
@@ -109,9 +109,29 @@ consecutive detections that should have been grouped to a single fixation (type
 if the gaze points are dispersed over 2 degrees due to bigger eye rotations, filtering with a maximum 
 dispersion of 1 degree could classify two separate fixations instead of one. Setting the maximum 
 dispersion too high might group fixations together that should have been considered separate fixations. 
-Ensure the maximum dispersion accommodates your viewing task. 
+Ensure the maximum dispersion accommodates your viewing task.
 
-## Sychronization
+## Blink Detector Thresholds
+In general, the default values for the [Blink Detector](/core/software/pupil-player/#blink-detector) should work reasonably 
+well on good quality eye data with robust pupil detection. However, it is often necessary to adjust the thresholds in the 
+event that [blinks](/core/terminology/#blinks) are not accurately  classified. Therefore, it is important to understand 
+the types of errors that can occur and to be able to spot them when they occur, thereby enabling you to make appropriate 
+adjustments.
+
+### Errors
+**A.** False negatives - Blinks are not being detected, e.g. due to the onset threshold being too high  
+**B.** False positives - The onset threshold is set too low and blinks are classified even though they did not occur  
+**C.** The end of a blink is not detected due to the offset threshold being too low. This can lead to erroneous blinks 
```

Comment:
```suggestion
**C.** The end of a blink is not detected due to the offset threshold being too high. This can lead to erroneous blinks 
```
Shouldn't this be too `high`, i.e. the offset _activation_ being too low and not able to reach the threshold?